src/cf.data.pre

```diff
@@ -10151,9 +10151,9 @@ DOC_START
 DOC_END
 
 NAME: hosts_file
-TYPE: string
+TYPE: SBuf
```

Comment:
AFAICT, this TYPE change (among other things) converts a previously required filename parameter to an optional one. Please do not do that.

FWIW, I recommend keeping Config.etcHostsPath (field and type) unchanged in this PR. There are other problems with parse/dump_SBuf() that we would be able to avoid solving in this PR if we remove that non-essential change from this PR.